core/feconf.py

```diff
@@ -209,7 +209,7 @@
 
 # The maximum number of results to retrieve in a datastore query
 # for suggestions.
-DEFAULT_SUGGESTION_QUERY_LIMIT = 100
+DEFAULT_SUGGESTION_QUERY_LIMIT = 1000
```

Comment:
Just confirming, I remember setting the limit based on feedback that the dashboard took too long to load. 10x the limit is a pretty drastic change. Just ensuring it is fine.

Reply:
Yup, the method for fetching data was O(N) before -- now it's O(1). The reduced limit is causing problems for reviewers, too.

Reply:
In that case, should we use the [old constant](https://github.com/oppia/oppia/pull/14164/files#diff-01f44d7b1b2d1be700ecfd1115d1dc9381f52404c1a334b0426e00e2f983d303L506) `DEFAULT_QUERY_LIMIT` (Which is also 1000)?

Reply:
I think we'll probably want to tune this further, so for now I would suggest leaving it as-is. What do you think?

Reply:
Yeah, I think that's fine! (Also, I think we fetch 1000 suggestions at a time but we only present 20 in a page (With navbar))

Reply:
Thanks! FYI, for reviews I think we only present 10 in a page (but currently, with 100 items, the whole thing just stops after 10 pages which is a problem).